imagej/imagej-ops2/src/main/java/net/imagej/ops2/types/ImgFactoryTypeExtractor.java

```diff
@@ -48,25 +49,21 @@
  *
  * @author Gabriel Selzer
  */
-public class ImgFactoryTypeExtractor implements TypeExtractor<ImgFactory<?>> {
+public class ImgFactoryTypeExtractor implements TypeExtractor {
 
-	@Override
-	public Type reify(final TypeReifier t, final ImgFactory<?> o, final int n) {
-		if (n != 0)
-			throw new IndexOutOfBoundsException();
-
-		return t.reify(o.type());
+	@Override public double getPriority() {
+		return Priority.NORMAL;
```

Comment:
Isn't `NORMAL` already the default? If not, shouldn't it be?

Reply:
Yeah, it should be. Fixed in 877d83f